### ISSUE TITLE
pkgjs/* and nodejs/ci-config* repo housekeeping

### ISSUE THREAD

**Issue by dominykas** [nodejs/package-maintenance#348]
I'm working on this - this is to keep track of things.

All repos need to have:

* [x] code of conduct
    * [x] PRs open
    * [x] Add at the repo root
* [x] contribution guidelines
    * [x] PRs open
* [x] a license file
    * [x] a matching entry in `package.json`
    * [x] https://github.com/pkgjs/dependents/blob/master/LICENSE#L189
* [x] authors file: details in https://github.com/nodejs/package-maintenance/issues/368
    * [x] PRs open
* [x] pull request template
* [x] is Apache OK https://github.com/pkgjs/dependents/blob/master/LICENSE?
* [x] A README.md which includes references to the GOVERNANCE.md in the package-maintenance repository as the authoritative governance which applies to the repository.
    * [x] PRs open

**Comment by jonchurch** [comment]
Perhaps it's more effective to use the [default community health file](https://help.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file) method of creating a .github repo where these standard documents live and are applied to all repos under the org?

I am not certain that it applies retroactively for existing repos, but that is my understanding.

**Comment by ljharb** [comment]
It will if they don't have the file in the repo already.

**Comment by dominykas** [comment]
Hmmm. That's obviously a lot more efficient than having them at the root of each repo, but is that sufficient from node.js org repo requirements perspective? Who would be able to get that done at node.js org level so that we don't have to do it in ci-config* repos?

**Comment by wesleytodd** [comment]
I think it is fine to have a hybrid to start.  So having them in some repos (ones which already have them) but then creating `.github` repo for the rest.  In the `pkgjs` org I think I would rather see us go this route so we don't go through this for every new repo.

**Comment by mhdawson** [comment]
Once we prove out the concept in the pkgjs org I think we should also explore doing it in the Node.js org as well. It think it would greatly increase the likelihood of new repos following the requirements.  But for now I'd just do the ci-* ones manually.

**Comment by bnb** [comment]
Huge +1 to using the .github repo approach and potentially exploring it for the nodejs GitHub org once we've seen it working effectively in pkgjs.

**Comment by dominykas** [comment]
OK, so here's https://github.com/pkgjs/.github/pull/1 - once merged, I'll see if my other PRs need an update (I think we should remove the per-repo files?) or close them.

**Comment by dominykas** [comment]
Adding to agenda, primarily to answer two questions:

1. Need approvals for https://github.com/pkgjs/.github/pull/1
2. Do we want to update existing repos to remove the existing templates?

**Comment by jonchurch** [comment]
One thing to consider that I don't think I mentioned anywhere, I believe that when using the default `.github` approach, those files are not included when the repo is cloned. I don't know if that matters at all, and I'd still prefer the `.github` approach, but it is a difference in behavior to note.

**Comment by dominykas** [comment]
OK, so what I did:

- CoC is included in `.github` repo, but also at the root of the repo (and removed from `.github` folder where it was present). While GH does handily provide a link to CoC when commenting/opening issues/PRs, I feel that the presence of the file at the root is important for additional visibility (and reassurance).
- I personally think that it's probably enough to have `CONTRIBUTING.md` in the `.github` repo, but the governance doc says it has to be in the repo, so I added it in the repo, at the root level, because I suspect it's more useful there, than hidden as a small link somewhere else.
- I kept the PR template in `.github` - I think that's good enough, and possibly each repo can add more details to it, but this satisfies the requirement?
